iex-observer.sh

```diff
@@ -45,7 +45,7 @@ fi
 # Trap exit so we can try to kill proxies that has stuck in background
 function cleanup {
   set +x
-  sudo sed -i '' "/${HOST_RECORD}/d" /etc/hosts
+  sudo sed -i "/${HOST_RECORD}/d" /etc/hosts
```

Comment:
I think this won't work on Mac :(

Reply:
Could you please try running `sed -i'' <pattern> <file>` on macOS. Note the absence of space between `-i` and `''`. it should work on macOS 10.9+.

Reply:
@alco It only works with space:
```
$ sed -i'' "/foo/d" e.txt 
sed: 1: "e.txt": invalid command code e
```

Reply:
Let's use this workaround then - https://stackoverflow.com/a/53408233/213682. I've pushed a new commit with the change.